tests/foreman/ui_airgun/test_operatingsystem.py

```diff
@@ -149,3 +79,72 @@ def test_positive_delete(session):
         assert session.operatingsystem.search(name)[0]['Title'] == os_full_name
         session.operatingsystem.delete(os_full_name)
         assert not session.operatingsystem.search(name)
+
+
+@tier3
```

Comment:
why t3? test steps aren't smth really complicated, imo it's rather integration than system level, i'd place `@tier2`.

Reply:
tiers has nothing to do with complexity
integration - we test how two components  working together and their interaction in between
here we have whole system gathered in one place and we test real end to end scenario